assets/js/pages/ClusterDetails/StoppedResources.test.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { screen } from '@testing-library/react';
+import '@testing-library/jest-dom';
+
+import { renderWithRouter } from '@lib/test-utils';
```

Comment:
You don't need `renderWithRouter` use `render` coming from `@testing-library/react`
```
import { screen, render } from '@testing-library/react';
```